projects/plugins/wpcomsh/wpcom-plugins/plugins.php

```diff
@@ -21,6 +21,9 @@ function wpcomsh_plugins_show_banner() {
 	if ( ! empty( get_option( 'is_fully_managed_agency_site' ) ) ) {
 		return;
 	}
+	if ( function_exists( 'current_user_has_wpcom_account' ) && ! current_user_has_wpcom_account() ) {
```

Comment:
@Addison-Stavlo Is this working? In my testing, I always see the banner even though I am logged in with a local Atomic user.

It seems, it can't find `current_user_has_wpcom_account()` function since it's defined in the mu-wpcom plugin, and maybe loaded later...

Reply:
Weird! This worked in testing the original PR, but maybe the load order is different in production or something?

Reply:
Im seeing the banner in my testing now as well...